src/pages/404.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
do we need both 404 pages? I thought I saw the second one ..

Reply:
@pavelgrinchenko ? ^^

Reply:
Do you mean `src/pages/404.js` and `src/components/404/`?
In first case it file for Gatsby which tells to it to generate page `404.html`. In the second case it's React component which will be rendered on this page, with all styles that it has. Other pages in the `src/pages' folder have the same structure.

Reply:
👍